app/services/phone_number_capabilities.rb

```diff
@@ -0,0 +1,32 @@
+class PhoneNumberCapabilities
+  VOICE_UNSUPPORTED_US_AREA_CODES = {
+    '648' => 'American Samoa',
+    '671' => 'Guam',
+    '670' => 'Northern Mariana Islands',
+    '340' => 'United States Virgin Islands',
+  }.freeze
+
+  attr_reader :phone
+
+  def initialize(phone)
+    @phone = phone
+  end
+
+  def sms?
+    true
+  end
+
+  def voice?
+    VOICE_UNSUPPORTED_US_AREA_CODES[area_code].nil?
+  end
+
+  def unsupported_location
+    VOICE_UNSUPPORTED_US_AREA_CODES[area_code]
+  end
+
+  private
+
+  def area_code
+    @area_code ||= Phony.split(Phony.normalize(phone, cc: '1')).second
```

Comment:
Alright, so this started breaking the spec for the IdV flow. The problem is that at this point the phone number has been formatted for the vendor and no longer includes the country code. It appears to still work for US numbers, but gets confused for numbers with a US country code.

Using Phony to normalize it to add the `+1` works, so I may do that.

Related: We may want to make sure we only accept US phone numbers for IdV in #1544